packages/devextreme/js/ui/drop_down_editor/ui.drop_down_editor.d.ts

```diff
@@ -86,6 +100,13 @@ export interface dxDropDownEditorOptions<TComponent> extends Omit<dxTextBoxOptio
      * @public
      */
     dropDownButtonTemplate?: template | ((buttonData: DropDownButtonTemplateDataModel, contentElement: DxElement) => string | UserDefinedElement);
+    /**
+     * @docid
+     * @default null
+     * @type object
```

Comment:
Is it required? It degrades types - [link](https://github.com/DevExpress/DevExtreme/pull/30723/files#diff-5b5792fb1577c537f54f5a3f6edaf6f5dc74cad5f5e408bb70568a2f1a137aa2R169)